tests/rbd/test_rbd_encryption.py

```diff
@@ -0,0 +1,314 @@
+from time import sleep
+
+from krbd_io_handler import krbd_io_handler
+
+from tests.rbd.rbd_utils import initial_rbd_config
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def device_cleanup(rbd, file_name, **kw):
+    """
+    Unmout the given file_name, unmap the given device_name encrypted using encryption_config
+    Args:
+        rbd: rbd object
+        file_name: <path>/<file> to be unmounted
+        kw:
+            "image_spec": image which was encrypted
+            "encryption_config": {<type>:<passphrase>} used for encryption
+            "device_type": default "nbd"
+            "device_name": device created for image map without encryption
+    """
+    flag = 0
+    if rbd.exec_cmd(
+        cmd=f"umount {file_name}",
+        sudo=True,
+    ):
+        log.error(f"Umount failed for {file_name}")
+        flag = 1
```

Comment:
Instead of `flag` we can directly `return 1` Is it?
It avoids next execution as well

Reply:
I don't want to avoid next execution. even if device unmount failed I would like to atleast unmap the device, since that will cleanup space for next tests to be executed. So adding flag there was done intentionally